### ISSUE TITLE
Session crashes with bibtex 0.4.2.1

### ISSUE THREAD

**Issue by neurotroph** [crsh/citr#57]
After upgrading my R packages I noticed that `citr` crashed as soon as I started opened the addin. This happened in both the repository version and the CRAN version. After some debugging I noticed that it is a problem with the latest version of `bibtex` published on CRAN (0.4.2.1), see https://github.com/romainfrancois/bibtex/issues/23

Downgrading `bibtex` to 0.4.2 solves the issue, e.g. by
```
devtools::install_version("bibtex", version="0.4.2", repos="http://cran.us.r-project.org")
```

Just posting it here, because some might run into the same issue.

**Comment by neurotroph** [comment]
Fixed in `bibtex` 0.4.2.2, now on CRAN (https://cran.r-project.org/web/packages/bibtex/index.html)

**Comment by crsh** [comment]
Thanks. I will make version 0.4.2.2 versioned dependency.
